### ISSUE TITLE
flux:composer color and border inconsistencies

### ISSUE THREAD

**Issue by ju5t** [livewire/flux#2159]
### Flux version

v2.7.0

### Livewire version

v4

### Tailwind version

v4

### Browser and Operating System

Chrome on MacOS

### What is the problem?

There are actually two issues.

- The focus ring color is set, this is not the case for any other Flux component. From what I can remember from previous issues, the focus ring wouldn't be something Flux would do something with. If `ring-blue-600` is the default, I think it should be for all components, not just this one. And I would prefer a more elegant color :)
- The border radius is set to `rounded-2xl`, all others have `rounded-lg`

### Code snippets to replicate the problem

```blade
<flux:composer wire:model="state.reply" placeholder="Send a message to the customer..."/>
<flux:textarea wire:model="state.reply" placeholder="Send a message to the customer..."/>
```

### Screenshots/ screen recordings of the problem

<img width="720" height="199" alt="Image" src="https://github.com/user-attachments/assets/e4e2a395-1413-4e1e-bd80-00122034e7b2" />

### How do you expect it to work?

I would expect it to be similar to other components.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@ju5t thanks for reporting! In my opinion, I probably wouldn't use the composer component in a form with other components. It's more used for a standalone input, say for a chat bot or something like that. You can see the documentation for it here https://fluxui.dev/components/composer

Not sure why you are getting the strange outline, in the docs, that doesn't seems to happen for me.

@calebporzio might have some thoughts on this as he just created this one.

**Comment by NimaKhadivar** [comment]
hi , outline is blue for me in composer component ( google chrome )

<img width="1021" height="105" alt="Image" src="https://github.com/user-attachments/assets/5f61bcdb-3608-4961-87df-d9a85dc7ccf8" />

but in safari its

<img width="891" height="91" alt="Image" src="https://github.com/user-attachments/assets/1543b8f0-5a9c-4035-97b5-87c18853972d" />

**Comment by ju5t** [comment]
@joshhanley that's kinda our use-case too, but we're using it to reply to an e-mail. It's a cool concept as we can inline file previews etc. But people will see it along side other components too, so it would be nice to have it behave similarly to a 'standard' textarea.

I'm not too bothered about the outline but it would be better if it's consistent for all components (border radius and color).

**Comment by calebporzio** [comment]
Good points. Just tagged `v2.7.1`

- I added a `variant="input"` to match the normal input component border radius
- I removed the custom focus styles (like we do with the rich text editor currently) so that we don't have to add custom styles that will clash with other inputs on various browsers. (There is no way to use a native focus ring around this component because it's not an actual focusable element (the text content within it is))

I agree that we should use custom focus styles across the board at some point, but that's a big lift so we'll just do this for now.

**Comment by ju5t** [comment]
@calebporzio got it, and yea no worries about the custom focus styling, that's not at all something I'm looking for at the moment. Thanks for the quick fix!
